src/widget/wsearchlineedit.cpp

```diff
@@ -77,6 +78,10 @@ WSearchLineEdit::WSearchLineEdit(QWidget* pParent, UserSettingsPointer pConfig)
     setInsertPolicy(QComboBox::InsertAtTop);
     setMinimumSize(0, 0);
     setSizeAdjustPolicy(QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon);
+    // Show available auto-complete suggestions in a popup.
+    // Without this (default: QCompleter::InlineCompletion) a suggestion is picked
+    // automatically, and needs to be removed manually if it doesn't fit
+    lineEdit()->completer()->setCompletionMode(QCompleter::PopupCompletion);
```

Comment:
This is a valuable change we should implement.
Though I'll revert this for now because
As soon as I find a way that allows to adequately style the additional drop-down I'll open a follow-up.